src/components/cart.lineitem.less

```diff
@@ -51,13 +52,24 @@
     font-weight: bold;
     padding-bottom: 3px;
     width: 60%;
+    vertical-align: top;
   }
 
   .options-col {
     display: inline-block;
     width: 50%;
   }
 
+  .promotions-col {
+    display: inline-block;
```

Comment:
I'd prefer keeping this fixed to the left of the container as below. 
![image](https://user-images.githubusercontent.com/39800563/49701028-9d819380-fbb4-11e8-80c6-043f8cb29ff9.png)

It's more mobile-friendly and wraps better dynamically in our containers. The current styles will break on mobile-tablet views. Also as the length of the product name increases, it will overlap with the width of this promotions div.